cmd/helm.go

```diff
@@ -187,8 +187,9 @@ var (
 	}
 	// This is also used in configure
 	helmHomeFlag = cli.StringFlag{
-		Name:  "helm-home",
-		Usage: "Home directory that is configured for accessing deployed Tiller server. (default: \"~/.helm\")",
+		Name:   "helm-home",
+		Usage:  "Home directory that is configured for accessing deployed Tiller server. (default: $HELM_HOME or \"~/.helm\")",
```

Comment:
```suggestion
		Usage:  "Home directory that is configured for accessing deployed Tiller server. You can also set this using the environment variable HELM_HOME. (default: \"~/.helm\")",
```

This is the verbiage we've been using for flags that can be set with env vars. Probably good to be consistent.